test/avrora/codec/plain_test.exs

```diff
@@ -186,10 +205,16 @@ defmodule Avrora.Codec.PlainTest do
     ~s({"namespace":"io.confluent","name":"Null_Value","type":"record","fields":[{"name":"key","type":"string"},{"name":"value","type":["null","int"]}]})
   end
 
+  defp map_json_schema do
+    ~s({"namespace":"io.confluent","name":"Map_Value","type":"record","fields":[{"name":"tested_field", "type": {"type": "map", "values": "string"}}]})
```

Comment:
May I suggest renaming `tested_field` into `map_field` since this example is a map specific